modules/raft/src/main/java/org/apache/ignite/internal/raft/configuration/RaftConfigurationSchema.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.raft.configuration;
+
+import org.apache.ignite.configuration.annotation.ConfigValue;
+import org.apache.ignite.configuration.annotation.ConfigurationRoot;
+import org.apache.ignite.configuration.annotation.ConfigurationType;
+import org.apache.ignite.configuration.schemas.table.VolatileRaftConfigurationSchema;
+
+/**
+ * Raft configuration schema.
+ */
+@SuppressWarnings("PMD.UnusedPrivateField")
+@ConfigurationRoot(rootName = "raft", type = ConfigurationType.LOCAL)
+public class RaftConfigurationSchema {
```

Comment:
And how now to configure for each table separately?

Reply:
Why do we need to configure raft differently for different tables?

Reply:
I meant that we need to configure **VolatileRaftConfigurationSchema** for different tables and how to do this is now not clear

Reply:
In this PR there is single volatile raft configuration for all volatile tables. We can change it later, right now (in main branch) we have raft configuration in table configuration, which is worse in my opinion